saleor/settings.py

```diff
@@ -567,3 +560,19 @@ def get_host():
 if REDIS_URL:
     CACHE_URL = os.environ.setdefault("CACHE_URL", REDIS_URL)
 CACHES = {"default": django_cache_url.config()}
+
+# Default True because storefront and dashboard don't support expiration of token
+JWT_DONT_EXPIRE = get_bool_from_env("JWT_DONT_EXPIRE", True)
```

Comment:
Not a big deal, but what about `JWT_EXPIRE` with default `False`? Sounds a bit more natural to me than a name with negation.